RadarSDK/Radar.m

```diff
@@ -538,8 +538,7 @@ + (void)logConversionWithNotification:(UNNotificationRequest *)request
                        deliveredAfter:(NSDate *)deliveredAfter {
     
     NSMutableDictionary *metadata = [[NSMutableDictionary alloc] initWithDictionary:request.content.userInfo];
-    NSDictionary<NSString *, NSString *> *result = [RadarUtils
-                                                    extractGeofenceIdAndTimestampFromIdentifier:request.identifier];
+    NSDictionary<NSString *, NSString *> *result = [RadarNotificationHelper extractMetadataFromNotificationIdentifier: request.identifier];
```

Comment:
Is it correct to be using the `request.identifier` for this? Are we liable to run into length issues? Wondering how much we can stuff in here.

Would be nice if we had some E2E validation of this, I'm worried about the campaign id getting cut off.